<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>1361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QProgressBar {
border-raidus: 16px;
background-color: blue
}

.QProgressBar:chunk {
border-radius: 16px;
background-color: red
}</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-14</x>
     <y>-10</y>
     <width>1381</width>
     <height>721</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QLabel {
background-color: black;
border-raidus: 10px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>140</y>
     <width>491</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>20th Century Font</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">.QLabel {
color: white;
}</string>
   </property>
   <property name="text">
    <string>WELCOME TO EINSTEIN</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>progressBar</zorder>
  <zorder>label_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
